roles/operator/README.rst

```diff
@@ -32,6 +32,11 @@ The default shell for the operator.
 
 A list of ssh authorized keys to add.
```

Comment:
Maybe reword this while at it?

`List of SSH public keys to add to the authorized keys for the operator account.`